.github/workflows/run-smokes.yml

```diff
@@ -9,17 +9,24 @@ jobs:
       router_versions: ${{ steps.router-versions.outputs.router_versions }}
       supergraph_versions: ${{ steps.supergraph-versions.outputs.supergraph_versions }}
     steps:
+      - uses: actions/checkout@v4
+        name: "Checkout rover repo"
       - run: |
-          JSON=$(curl -L -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" https://api.github.com/repos/apollographql/router/releases | jq -rc '[.[] | select((.name | contains("-") | not ) and (.name | contains("v")))] | [.[:3].[].name  | sub("v";"")]')
+          npm install -g semver
+        name: "Install `semver` cli"
+      - run: |
+          ls -al
+          JSON=$(source get_latest_x_versions.sh 3 apollographql router router latest-1 1)
```

Comment:
__Nit__: why use `source` over `./`?

Reply:
So I tried using `./` but because you're running it in a subshell you have to us `source` otherwise the interpreter gets confused